src/hooks/useCanInteract.jsx

```diff
@@ -1,6 +1,11 @@
 import { useTX } from '../contexts/TXContext';
 
-const defaults = ['isConnected', 'isSameChain', 'isMember'];
+const defaults = [
+  'isConnected',
+  'isSameChain',
+  'isMember',
+  'isMemberSpamFilter',
```

Comment:
Would this activate the check 'isMemberSpamFilter' even if the spam filter is not enalbed? 

And would there be any benefit of breaking up 'isMember' and 'spamFilter' since we already have the isMember check?

Reply:
yep agree on both points. adjusting this.